assets/base.css

```diff
@@ -1428,6 +1428,10 @@ button.shopify-payment-button__button--unbranded {
   display: flex;
 }
```

Comment:
Good catch. That's because the `modal-dialog` is checking for a `nodeName` = `modal-dialog`, but this is a custom element that has a different node name (`quick-add-modal`). I'll update it to check to compare the target to the element instead :P